netlify/netlify-plugin-nextjs
# chore: add e2e

### Summary
Add tests for the plugin. Tests for other demo sites (basepath, nx, and more) will follow

- [x] Default site
- [x] Preview mode
- [x] Headers
- [x] Rewrites/redirects
- [x] Trailing slash
- [x] Catch-all/optional catch-all, getxxprops
- [x] Dynamic imports
- [x] Custom 404/500
- [x] Static assets in `public`
- [x] cookie header locale issue

### Test plan
Visit deploy logs to see it being run with the plugin otherwise:

1. Checkout this branch
2. Run `npm i`
3. Run `npm run cy:open`
4. Run all tests
5. Happy

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal (small picture)
https://github.com/netlify/pod-ecosystem-integrations/issues/33

### Standard checks:


---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
